SoftLayer/CLI/loadbal/protocol_add.py

```diff
@@ -0,0 +1,66 @@
+"""Add a new load balancer protocol."""
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import exceptions
+from SoftLayer.CLI import formatting
+
+
+# pylint: disable=unused-argument
+def parse_proto(ctx, param, value):
+    """Parses the frontend and backend cli options"""
+    proto = {'protocol': 'HTTP', 'port': 80}
+    splitout = value.split(':')
+    if len(splitout) != 2:
+        raise exceptions.ArgumentError("{}={} is not properly formatted.".format(param, value))
+    proto['protocol'] = splitout[0]
+    proto['port'] = int(splitout[1])
+    return proto
+
+
+@click.command(cls=SoftLayer.CLI.command.SLCommand, )
+@click.argument('identifier')
+@click.option('--frontend', '-f', required=True, default='HTTP:80', show_default=True, callback=parse_proto,
+              help='PROTOCOL:PORT string for incoming internet connections.')
+@click.option('--backend', '-b', required=True, default='HTTP:80', show_default=True, callback=parse_proto,
+              help='PROTOCOL:PORT string for connecting to backend servers.')
+@click.option('--method', '-m', help="Balancing Method.", default='ROUNDROBIN', show_default=True,
+              type=click.Choice(['ROUNDROBIN', 'LEASTCONNECTION', 'WEIGHTED_RR']))
+@click.option('--session', '-s', required=True,
+              help="Session stickiness type. Valid values are “SOURCE_IP” “HTTP_COOKIE”")
+@click.option('--max', help="Use a Public to Public loadbalancer.")
```

Comment:
```suggestion
@click.option('--max', help="Max Connections setting", type=Click.Int)
```
Fix the help text, and make it a Int type